controller/src/main/java/org/jboss/as/controller/PersistentResourceXMLDescriptionReader.java

```diff
@@ -17,7 +17,9 @@
 /**
  * An {@link XMLElementReader} based on a {@link PersistentResourceXMLDescription}.
  * @author Paul Ferraro
+ * @deprecated Superseded by {@link org.jboss.as.controller.persistence.xml.SubsystemResourceXMLElementReader}.
  */
+@Deprecated(forRemoval = true)
```

Comment:
Hello @pferraro Do not forget our policy about adding the "since" attribute to any deprecated method. thanks!

Reply:
Fixed.